datacite/schema/example.json

```diff
@@ -0,0 +1,117 @@
+{
+    "identifier": {
+        "identifier": "10.5072/example-full",
+        "type": "DOI"
```

Comment:
I think it's best if we keep as close as possible to DataCite's naming - i.e. ``identifierType`` instead of ``type``.